packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -424,24 +425,28 @@ export class LitDevSearch extends LitElement {
   private _onKeydown(e: KeyboardEvent) {
     switch (e.key) {
       case 'ArrowDown':
+        this._closedByEscape = false;
         this._selectNext();
         break;
       case 'ArrowUp':
+        this._closedByEscape = false;
         this._selectPrevious();
         break;
       case 'Enter':
         this._select();
         break;
       case 'Escape':
         const oldText = this._searchText;
-        this._searchText = '';
-        this._suggestions = [];
+        this._closedByEscape = true;
         this._selectedIndex = -1;
         // prevent the input from closing drawer if there was text
         if (oldText.trim()) {
           e.stopPropagation();
         }
         break;
+      default:
+        this._closedByEscape = false;
```

Comment:
Could we do this in a `blur` handler?

Reply:
a11y model for combobox is that the input is never un-focused even as you navigate up and down. And if you have a native input that you hit `escape` it will not blur the input. Though, the menu still closes if the input is blurred.

Reply:
did blur anyway